src/mudlet.cpp

```diff
@@ -1859,7 +1859,6 @@ bool mudlet::echoWindow(Host* pHost, const QString& name, const QString& text)
 bool mudlet::echoLink(Host* pHost, const QString& name, const QString& text, QStringList& func, QStringList& hint, bool customFormat)
 {
     QMap<QString, TConsole*>& dockWindowConsoleMap = mHostConsoleMap[pHost];
```

Comment:
What is even more confusing is that `dockWindowConsoleMap` ___is nothing to do with_ dockable _windows in this method___ - IIRC it is a QMap of the `TConsoles` that can be laid out on-top of the main `TConsole` instance!

As for the redundant `t` it probably comes about  because the code that was copy and pasted worked on a different `QMap` where the key was a `std::string` instead of a `QString`...!